src/javascript/static/pages/responsible_trading.js

```diff
@@ -0,0 +1,26 @@
+const State          = require('../../_common/storage').State;
+const Client         = require('../../app/base/client');
+const BinarySocket   = require('../../app/base/socket');
+const getElementById = require('../../../javascript/_common/common_functions').getElementById;
+
+const ResponsibleTrading = (() => {
+
+    const onLoad = () => new Promise(resolve => {
+        BinarySocket.wait('landing_company').then(() => {
+            const current_landing_company_shortcode = State.getResponse('authorize.landing_company_name') || 'default';
```

Comment:
I don't think we need a fallback for this.
```suggestion
            const current_landing_company_shortcode = State.getResponse('authorize.landing_company_name');
```

Reply:
also you can just use `Client.get('landing_company_shortcode')`

Reply:
and we should fallback to `State.getResponse('landing_company.gaming_company.shortcode')` for logged clients

Reply:
corrected